src/index.html

```diff
@@ -20,8 +20,389 @@
       href="styles/index.scss"
```

Comment:
The stylesheet link references a SCSS file. Ensure that this SCSS file is compiled to a CSS file, as browsers cannot directly interpret SCSS.